src/operations-tracker-header/operations-tracker-header.tsx

```diff
@@ -54,14 +50,14 @@ export const OperationsTrackerHeader = React.memo(
               </Button>
             )}
           </div>
-          <div>
-            <Search onSearchChange={debouncedFilter} />
-          </div>
         </div>
         {openDescription && (
           <div className={classes.description}>
-            It monitors changes in cache, fired mutations and
-            activated/deactivated queries.
+            <span>
+                It monitors changes in cache, fired mutations and
+                 activated/deactivated queries.
+            </span>
+            <Link href="https://github.com/jpsahoo14/apollo-inspector-ui" inline rel="noopener noreferrer"> Read Me </Link>
```

Comment:
This should be the link

https://github.com/jpsahoo14/apollo-inspector-ui/blob/main/extension/readme.md

Reply:
Changes

Reply:
I still see the old url